### PR TITLE
Make camera's translation speed scale relatively to its Zoom Level.

### PR DESCRIPTION
**Pull request by 3x3y3z3t**
#### Describe the issue (if no issue has been made)
This PR doesn't fix any issue.
This PR will improve the existing `OrthographicCameraController` described below.

#### PR impact _(Make sure to add [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords))_
List of related issues/PRs this will solve:

 Impact                  | Issue/PR
------------------------ | ------
Other PRs this solves    | #171 (which has been closed)<br>resolves #161 

#### Proposed fix _(Make sure you've read [on how to contribute](https://github.com/TheCherno/Hazel/blob/master/.github/CONTRIBUTING.md) to Hazel)_
This PR adds/improves the following:
- OrthographicCameraController:
  - Camera translation speed now scales with the zoom level (implementation is different from #161).
  - Zoom Level and Enable Rotation can now be manually set at any time/any place.
  - There is a min/max zoom level limitation, but manually setting zoom level using `SetZoomLevel()` overrides that limitation.

#### Additional context
None.

### CODE REVIEW

File: Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp [hunk 1]
```diff
@@ -54,7 +54,18 @@ namespace Hazel {
 
 		m_Camera.SetPosition(m_CameraPosition);
 
-		m_CameraTranslationSpeed = m_ZoomLevel;
+		//m_CameraTranslationSpeed = m_ZoomLevel;
+	}
+
+	void OrthographicCameraController::SetZoomLevel(float level)
+	{
+		HZ_CORE_ASSERT(level > 0.0f, "Zoom Level have to be bigger than 0.0f!");
```
Comment (Emilundpixeln) [inline Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp]: Zoom Level has to be bigger than 0.0f!

Comment (3x3y3z3t) [reply Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp]: I've updated that in the local repo.

Comment (LovelySanta) [reply Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp]: Update the PR as well?

Comment (3x3y3z3t) [reply Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp]: I wanted to wait for other change suggestions and update them all at once.
Anyway, PR is updated.

**Comment by TheCherno** [comment]
Closed (obsolete).

**Pull request closed without merging**